src/Hosting/Hosting/test/Internal/HostingEventSourceTests.cs

```diff
@@ -216,5 +263,36 @@ protected override void OnEventWritten(EventWrittenEventArgs eventData)
                 }
             }
         }
+
+        private class CounterListener : EventListener
```

Comment:
This could be useful for testing all counters. We'll have to remember this when adding other counters and promote this to `src/Shared` (or you could just do it now...)

Reply:
Or maybe add this to https://github.com/aspnet/Extensions/blob/master/src/TestingUtils/Microsoft.AspNetCore.Testing/src/Tracing/EventSourceTestBase.cs ?

Reply:
Yep. I have a thread with corefx about how to test these things. As I add more counters I'll move it to shared.